devtools/docker-compose.yml

```diff
@@ -177,34 +166,33 @@ services:
     build:
       context: ../src/backend
       dockerfile: Dockerfile.imap-worker
-    image: caliopen_imap_worker
     links:
       - cassandra:cassandra
       - object-store:minio
       - elasticsearch:elasticsearch
       - nats:nats
     volumes:
       - ../src/backend/configs:/etc/caliopen
+      - /etc/ssl/certs:/etc/ssl/certs
```

Comment:
This is a temporary workaround and should not be pushed, we risk never putting in place a real solution. Containers that need certificates need to include the certs.